src/ctap/hid/mod.rs

```diff
@@ -98,21 +99,15 @@ impl CtapHid {
     const COMMAND_KEEPALIVE: u8 = 0x3B;
     const COMMAND_ERROR: u8 = 0x3F;
     // TODO: optional lock command
-    const COMMAND_LOCK: u8 = 0x04;
```

Comment:
Can be left in to reduce diff, will be refactored later!

Reply:
I think this comment is invalid after I rebased to the head of develop on Monday.